normandy/settings.py

```diff
@@ -483,3 +497,4 @@ class Test(Base):
     AUTOGRAPH_URL = None
     AUTOGRAPH_HAWK_ID = None
     AUTOGRAPH_HAWK_SECRET_KEY = None
+    OIDC_USER_ENDPOINT = 'https://auth.example.com/userinfo'
```

Comment:
Is this a duplicate? It seems like it unconditionally overrides the entire `CORS` class's contribution.

Reply:
Not sure what you mean. The point of this is to make sure the tests never actually reach out to auth.mozilla.auth0.com. This way the tests aren't at the mercy of the environment. 

Also, the domain auth.example.com will not resolve so if your unit tests accidentally *don't* mock requests, you'd find out immediately.

Reply:
I totally misread this on my first pass. I thought this class was the base settings, and missed that it was the test settings. It is good as is, Sorry!